Cargo.toml

```diff
@@ -23,7 +23,6 @@ name = "Emulsion"
 identifier = "io.github.arturkovacs.emulsion"
 icon = ["resource_dev/emulsion.png"]
 version = "1.9"
```

Comment:
I'm sorry my previous comment was ambiguous. I meant to ask if it's possible to remove this version so that the build system takes the version number from above.